django_querysets_single_query_fetch/service.py

```diff
@@ -42,6 +42,112 @@ class QuerysetsSingleQueryFetch:
     def __init__(self, querysets: list[QuerySet]) -> None:
         self.querysets = querysets
 
+    def _get_fetch_count_compiler_from_queryset(
+        self, queryset: QuerySet, using: str
+    ) -> Any:
+        """
+        slightly modified copy paste from get_count and get_aggregation in django.db.models.sql.compiler
+        """
+        obj = queryset.query.clone()
+        obj.add_annotation(Count("*"), alias="__count", is_summary=True)
+        added_aggregate_names = ["__count"]
+        if not obj.annotation_select:
+            return {}
```

Comment:
is this handled?

Reply:
Removed ,no need for this. This will never occur.